bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_DisappTrks/C1C1_C1N1_2j_proc_card.dat

```diff
@@ -0,0 +1,43 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.3                 2012-11-01         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+import model mssm
+define p = p b b~
+define j = j b b~
```

Comment:
We do not have any preference for the 4 flavor scheme, so if the 5 flavor scheme is the default in CMS, we should use that.  I can switch to "import model mssm-no_b_mass" but I'm not sure which pdfs and maxjetflavour to use.  Are there any examples using this model I can follow?  I couldn't find any in our repository using this command:
genproductions/bin/MadGraph5_aMCatNLO/cards/production/13TeV]$ grep -r "mssm-no_b_mass" * 
Thanks for your help with this!